Update database schema docs (#231)
kubearchive/kubearchive: docs/modules/ROOT/pages/reference/database.adoc

|varchar not null
|Version of the resource (`metadata.resourceVersion`).

|created_at
|timestamp not null

> |Timestamp when the record is inserted in this table

Reviewer: For consistency, period at the end of the description.

```suggestion
|Timestamp when the record is inserted in this table.
```

Author: Fixed!

|updated_at
|timestamp not null

> |Last time the record of this table was updated.

Reviewer 2: ```suggestion
|Last time a record in this table was updated.
```

Author: Fixed!
